docs/install/from_source.rst

```diff
@@ -323,7 +323,7 @@ like ``virtualenv``.
 
    .. code:: bash
 
-       pip3 install --user tornado psutil xgboost cloudpickle
+       pip3 install --user tornado psutil xgboost==1.5.0 cloudpickle
```

Comment:
Thanks for the fix, note that we can allow any version of xgboost before 1.6.0, we can allow a larger range.
```suggestion
       pip3 install --user tornado psutil  'xgboost<1.6.0' cloudpickle
```